projects/eval-adxl367z/builds.json

```diff
@@ -0,0 +1,10 @@
+{
+      "xilinx": {
+            "demo": {
+                  "flags": "DUMMY_EXAMPLE",
```

Comment:
`"flags" : "NEW_CFLAGS=-DDUMMY_EXAMPLE",`
otherwise, it won't work :)

Reply:
This is what I used for stm32 platform:
```
{
  "stm32": {
    "dummy_example": {
      "flags" : "DUMMY_EXAMPLE=y IIO_EXAMPLE=n"
    },
    "iio_example": {
      "flags" : "DUMMY_EXAMPLE=n IIO_EXAMPLE=y"
    }
  }
}
```
I would suggest using something like this (seeing how we are using y or n values):

```
{
  "xilinx": {
    "dummy_example": {
      "flags" : "DUMMY_EXAMPLE=y",
     ...
    }
}
```

Reply:
I guess both approaches are valid:
For example, cn0561 (recent project merged into the master branch):
https://github.com/analogdevicesinc/no-OS/blob/master/projects/cn0561/builds.json